frontend/packages/dsco/src/divider/divider.module.scss

```diff
@@ -0,0 +1,39 @@
+@import '@/common/styles/primitiveColors.css';
+
+// Divider common styles
+hr.divider {
+  border: 0;
+  border-top: 1px solid;
+  margin: 0;
+  width: 100%;
+}
+
+// Divider Colors
+hr.divider-color {
+  &-primary {
+    border-top-color: var(--neutral-gray-20);
```

Comment:
I see that in Figma (DSCO components file) we already have semantic colors for Divider
If possible, let's use them in code from the beginning as well please. Though this can also be done in a separate follow up pr as well